### PR TITLE
feat(service): unify timeouts

### PR DESCRIPTION
**Pull request by Serpentiel**
## About this change—what it does

this PR unifies the timeout values across all resources so that they are managed in a more obvious way, and uses a constant value so we are able to easily adjust it when/if needed

this PR also keeps an ability for users to manage the timeout values for their resources, and also adds `Default` timeout option to all the resources where it was missing, as well as all the other CRDU timeouts

this PR is linked to and was discovered during #1054

## Why this way

we want to allow our users to manage their timeouts, while both having a reasonable default timeout value, easily modifiable if needed, and unified across all the resources so they become functioning in a more obvious manner for the end users

### CODE REVIEW

**Review by byashimov — APPROVED** [review]

File: internal/schemautil/service.go [hunk 1]
```diff
@@ -16,6 +16,9 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+// DefaultTimeoutMinutes is the default timeout for service operations.
+const DefaultTimeoutMinutes = 20
```
Comment (ivan-savciuc) [inline internal/schemautil/service.go:20]: By the var name I would naturally expect it to be of type `time` with some timeout

**Review by ivan-savciuc — DISMISSED** [review]
LGTM

File: internal/schemautil/service.go (hunk 1 above)
Comment (Serpentiel) [reply internal/schemautil/service.go]: this has been changed completely

**Review by byashimov — APPROVED** [review]

File: internal/schemautil/service.go (hunk 1 above)
Comment (Serpentiel) [reply internal/schemautil/service.go]: please check it again, I moved it to a function completely, also I checked with other open source projects, most use `Minutes` suffix when denoting an integer constant

**Comment by byashimov** [comment]
lgtm

**Review by ivan-savciuc — APPROVED** [review]
LGTM

**Pull request merged**